src/vos/tests/evt_ctl.c

```diff
@@ -1223,6 +1223,134 @@ test_evt_iter_delete(void **state)
 	assert_int_equal(rc, 0);
 }
 
+static void
+test_evt_find_internal(void **state)
+{
+	struct test_arg		*arg = *state;
+	daos_handle_t		 toh;
+	daos_handle_t		 ih;
+	struct evt_entry_in	 entry = {0};
+	struct evt_entry	 *ent;
+	struct evt_rect		 rect;
+	struct evt_entry_array	 ent_array;
+	bio_addr_t		 addr;
+	int			 rc;
+	int			 epoch;
+	int			 offset;
+	int			 hole_epoch;
+	char testdata[] = "deadbeef";
+
+	/* Create a evtree */
+	rc = evt_create(EVT_FEAT_DEFAULT, 13, arg->ta_uma, arg->ta_root,
```

Comment:
/** Valid tree order */
enum {
        EVT_ORDER_MIN                   = 4,
        EVT_ORDER_MAX                   = 128,
};
evt order can be any value between 4 to 128. I used 13 since most of the tests in evt_ctl.c use the same number...